mesh/pmesh.hpp

```diff
@@ -148,6 +145,53 @@ class ParMesh : public Mesh
 
    bool WantSkipSharedMaster(const NCMesh::Master &master) const;
 
+   /// Fills out partitioned Mesh::vertices
+   int BuildLocalVertices(const Mesh& global_mesh, const int *partitioning,
+                          Array<int> &vert_global_local);
+
+   /// Fills out partitioned Mesh::elements
+   int BuildLocalElements(const Mesh& global_mesh, const int *partitioning,
+                          const Array<int> &vert_global_local);
+
+   /// Fills out partitioned Mesh::boundary
+   int BuildLocalBoundary(const Mesh& global_mesh, const int *partitioning,
+                          const Array<int> &vert_global_local,
+                          Array<bool>& activeBdrElem,
+                          Table* &edge_element);
+
+   void FindSharedFaces(const Mesh &mesh, const int* partition,
+                        Array<int>& face_group,
+                        ListOfIntegerSets& groups);
+
+   int FindSharedEdges(const Mesh &mesh, const int* partition,
+                       Table* &edge_element, ListOfIntegerSets& groups);
+
+   int FindSharedVertices(const Mesh &mesh, const int *partition,
+                          Table* vertex_element,
+                          ListOfIntegerSets& groups);
```

Comment:
The parameter `mesh` is not used in the implementation -- it can be removed.

Reply:
Removed.